test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -3,8 +3,11 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import java.util.LinkedList;
```

Comment:
It would be good to sort import statements using lexicographical order. This is so that easily identify what you imported/did not import when there are large number of import statements. 

If I'm not mistaken, in higher levels of AddressBook there should be an automatic check which will alert you about this. (More details about this will be made clearer when you progress towards the build automation part)